tests/scripts/thread-cert/node.py

```diff
@@ -1897,16 +1897,41 @@ def disable_br(self):
         self.send_command('br disable')
         self._expect_done()
 
-    def get_omr_prefix(self):
+    def get_br_omr_prefix(self):
         cmd = 'br omrprefix'
         self.send_command(cmd)
         return self._expect_command_output()[0]
 
-    def get_on_link_prefix(self):
+    def get_netdata_omr_prefixes(self):
+        prefixes = [prefix.split(' ')[0] for prefix in self.get_prefixes()]
+        return prefixes
+
+    def get_br_on_link_prefix(self):
         cmd = 'br onlinkprefix'
         self.send_command(cmd)
         return self._expect_command_output()[0]
 
+    def get_netdata_on_link_prefixes(self):
```

Comment:
```suggestion
    def get_netdata_non_nat64_prefixes(self):
```

The code checks that the route isn't NAT64. But we actually don't know that it must be an on-link prefix.
There may also be other routes:
- OMR prefixes of other Thread networks
- Manually added routes (e.g. from user)

Reply:
Updated. Thanks!